docs/docs.rst

```diff
@@ -0,0 +1,37 @@
+==========================
+Updating The Documentation
+==========================
+
+You need to have Python to build the documentation locally.
+
+The documentation is built in our continuous integration every time
+if you feel bold you can just edit these ``.rst`` files and see if the
+pull request succeeds to build.
+
+To build the documentation, create and activate a Python virtualenv.
+For exampe:
+
+.. code-block:: shell
+
+    $ virtualenv -p `which python3.6` .venv
+    $ source .venv/bin/activate
```

Comment:
It's been a loooooong time since I last wrote any python, is virtualenv still the way to go? It seems I'm hearing more and more about pipenv.

Reply:
pipenv is cool but not importantly different.